features/polyExplorer/src/screens/entityDetails/companyRevenueChart/companyRevenueChart.css

```diff
@@ -49,8 +49,8 @@
     background-color: #3ba6ff;
     width: 12%;
     max-width: 80px;
-    bottom: 24.5%;
-    z-index: 200;
+    bottom: 15.5%;
```

Comment:
OK, this is weird, but I guess you've tested it... Still, I think it's one of the few with relative size?

Reply:
Yes, tbh I just sticked to the given unit, the tests were fine